content/en/blog/2019/v1beta1-authorization-policy/index.md

```diff
@@ -0,0 +1,283 @@
+on services using three configuration resources: `ClusterRbacConfig`, `ServiceRole` and `ServiceRoleBinding`.
+With this API, users have been able to enforce access control in mesh-level, namespace-level and
+service-level. Like any other RBAC policy, Istio RBAC uses the same concept of role and binding for granting
+permissions to identities.
+
+While this functionality has proven to be working for many customers, we have also recognised
+some shortcomings in its design that is causing many confusions and preventing its further adoption.
+
+For example, some users have had the wrong impression that the enforcement happens in service-level because
+`ServiceRole` uses service to specify where to apply the policy, however, the policy is actually applied
+in workload-level, the service is only used to find the corresponding workload. This has some big impact, especially
+when multiple services are referring to the same workload. A `ServiceRole` for service A will also affect
+service B if the two services are referring to the same workload, this surprises customers and could even cause
+security vulnerability.
+
+The other example is it's a little bit hard to maintain and manage the Istio RBAC configurations because three
+are three different customer resource definitions in Istio RBAC. Istio has already had an issue that there
+are too much CRDs to learn and manage, and the learning and maintenance burden of having three different configurations
+in Istio RBAC turned out to be greater than its benefit.
+
+To address these, and other issues, Istio 1.4 introduces the `v1beta1` authorization policy which will
+completely replace the previous `v1alpha1` RBAC policy going forward. The `v1beta1` policy provides
+more features but is not backward compatible and will require manual conversion from the old API (we
+will provide tool to help automate the conversion).
+
+In this article, we will introduce the new `v1beta1` authorization policy, talk about its design principle
+and the migration from the `v1alpha1` RBAC policy.
+
+## Design principles
+
+A few key design principles played a role in the `v1beta1` authorization policy:
+
+* Align with [Istio Configuration Model](https://goo.gl/x3STjD). The configuration model
+provides a unified way for configuration hierarchy, resolution, target selection and etc.
+
+* Improve user experience by simplifying the API. It's much easier to manage one custom resource
+definition (CRD) that includes all access control specification instead of looking at multiple
+different CRDs.
+
+* Make policy semantics more flexible without introducing much complexity. For example, it's
+very useful to apply the policy to the edge (ingress/egress gateway) to enforce access control
+for all traffic entering/exiting the mesh.
+
+## Authorization Policy
+
+An `AuthorizationPolicy` enables access control on workloads. This section gives
```

Comment:
Can we link to the docs for auth policy here?

Reply:
We should but the reference page hasn't been updated to include the new authz policy, I will check with @geeknoid and add the link when it's done.

Reply:
The reference docs are up to date at this point. If you're not seeing the content you expect, then the problem is probably in the api repo. In the protos, make sure the content isn't being hidden with $hide annotations, and that the proto has got a $location annotation. Without that, the doc will not be installed in the site.

Note that we're pulling docs out of the release-1.4 branch of istio/api at the moment, so any changes would need to be applied in that branch.

Reply:
Updated.